pkg/controllers/provisioning/scheduling/scheduling_benchmark_test.go

```diff
@@ -115,7 +115,8 @@ func benchmarkScheduler(b *testing.B, instanceCount, podCount int) {
 	provisioner := test.Provisioner(test.ProvisionerOptions{Limits: map[v1.ResourceName]resource.Quantity{}})
 
 	instanceTypes := fake.InstanceTypes(instanceCount)
-	cloudProv := &fake.CloudProvider{InstanceTypes: instanceTypes}
+	cloudProv := fake.CloudProvider()
```

Comment:
```suggestion
	cloudProv := fake.NewCloudProvider()
```

Is this call correct?

Reply:
Nope, don't know how that got through. Thanks!